data_example_test.go

```diff
@@ -109,6 +109,15 @@ func (vs strs) CopyNTimes(from ep.Data, fromRow, toRow int, duplications []int)
 	}
 }
 
+func (vs strs) CopyByIndexes(from ep.Data, fromRows []int, toRow int) {
+	src := from.(strs)
+
+	for _, n := range fromRows {
```

Comment:
to clarify this var, what do you think about:
```suggestion
	for _, idx := range fromRows {
```